source/clear-linux/guides/maintenance/mixer.rst

```diff
@@ -1,871 +1,842 @@
+#. Reload the Docker daemon:
 
-Next, set the `Version` field to the mix version that you want the content
-mixer to use to build the image. `ister` allows you to build an image from
-any mix version that you have built, not just from the current version. In
-our example so far, `Version` is set to 10.
+   .. code-block:: bash
 
-With the `ister` tool configured, build the image with the following command:
+      sudo systemctl daemon-reload
 
-.. code-block:: bash
+Configure the Docker container proxies, in order to pass proxy
+settings to containers:
 
-   sudo mixer build image
+#. Create a directory for your container config:
 
-Mixer automatically looks for the :file:`release-image-config.json` file, but
-you can freely choose the filename. To use a different name, simply pass the
-*--template* flag when creating your image, for example:
+   .. code-block:: bash
 
-.. code-block:: bash
+      mkdir ~/.docker
 
-   sudo mixer build image --template path/to/file.config
+#. Create the config file :file:`~/.docker/config.json` and add the following
+   entries, using your own proxy values:
 
-By default, `ister` uses the format version of the build machine it runs on.
-Therefore, if the format you are building differs from the format of the |CL|
-OS you are building on, you must use the *--format <FORMAT_NUMBER>*
-flag. Find the current format version of your OS with the following command:
+   .. code-block:: console
 
-.. code-block:: bash
+      {
+        "proxies":
+        {
+          "default":
+          {
+            "httpProxy": ":",
+            "httpsProxy": ":"
+          }
+        }
+      }
 
-   sudo cat /usr/share/defaults/swupd/format
+#. Set ownership and permission on the docker config directory:
 
-Update the next mix version information
-=======================================
+   .. code-block:: bash
 
-Increment the mix version number for the next mix with the following command:
+      sudo chown "$USER":"$USER" /home/"$USER"/.docker -R
+      sudo chmod g+rwx "$HOME/.docker" -R
 
-.. code-block:: bash
+Lastly, configure proxies to allow mixer to access upstream content from behind
+a firewall. For example:
 
-   mixer versions update
+#. Open your :file:`$HOME/.bashrc` file and add proxy and port values for the
+following:
 
-This command automatically updates the mix version stored in the
-:file:`mixversion` file, incrementing it by 10. To increment by a different
-amount, use the *--increment* flag, for example:
+   .. code-block:: console
 
-.. code-block:: bash
+      export http_proxy="<proxy-url>:<port>"
+      export https_proxy="<proxy-url>:<port>"
+      export HTTP_PROXY="<proxy-url>:<port>"
+      export HTTPS_PROXY="<proxy-url>:<port>"
+      export no_proxy="<...>"
 
-   mixer versions update --increment 10
+#. Log out and log back in for the proxies to take effect.
 
-Alternatively, to set the mix version to a specific value, use the
-*--mix-version* flag, for example:
+.. rst-class:: content-collapse
```

Comment:
What is this line?  I see a code-block that shows ".. rst-class:: content-collapse"

Reply:
This doesn't get rendered when built with Sphinx, and it provides additional functionality when built against our custom read the docs theme.